controller/messages.py

```diff
@@ -0,0 +1,39 @@
+def amountMessage(status):
+    if(status == "Paid"):
+        return 'What was the amount spent?'
```

Comment:
fetch each of the strings from a <language>-msgs.json file or default.json (all placed inside language folder).

env language config will tell us which file is needed to be used. If not configured should use default-messages.json